apps/passport-server/src/services/telegramService.ts

```diff
@@ -400,7 +369,12 @@ export class TelegramService {
       },
       validEventIds: {
         argumentType: ArgumentTypeName.StringArray,
-        value: undefined,
+        // For local development, we do not validate eventIds
+        // If you want to test eventId validation locally, copy the `id` field from `pretix_events_config`
+        // and add it to ALLOWED_EVENT_IDS. Then set value: ALLOWED_EVENT_IDS.map((e) => e.eventId)
+        value: isLocalServer()
+          ? undefined
+          : ALLOWED_EVENT_IDS.map((e) => e.eventId),
```

Comment:
nit: can we rename to ALLOWED_EVENTS` since it contains both the id and the name?